lib/rpmsg/rpmsg_virtio.c

```diff
@@ -315,6 +315,8 @@ static void rpmsg_virtio_release_rx_buffer(struct rpmsg_device *rdev,
 	/* Return buffer on virtqueue. */
 	len = virtqueue_get_buffer_length(rvdev->rvq, idx);
 	rpmsg_virtio_return_buffer(rvdev, rp_hdr, len, idx);
+	/* tell peer we return some rx buffer */
```

Comment:
@GUIDINGLI 
Please, could you fix typo before I merge?
s/ tell peer we return some rx buffer / **T**ell peer we return some rx buffer**s**/